qemu/qemu.go

```diff
@@ -1513,6 +1497,26 @@ func (config *Config) appendKnobs() {
 			config.qemuParams = append(config.qemuParams, numaMemParam)
 		}
 	}
+}
+
+func (config *Config) appendKnobs() {
+	config.appendMemoryKnobs()
+
+	if config.Knobs.NoUserConfig == true {
```

Comment:
Doesn't this change the order that the parameters will appear on the qemu command line?  Calling config.appendMemoryKnobs after if config.Knobs.Daemonize == true { would preserve the current order.